test/Microsoft.Dnx.Runtime.Tests/project.json

```diff
@@ -2,7 +2,6 @@
     "dependencies": {
         "Microsoft.AspNet.Testing": "1.0.0-*",
         "Microsoft.Dnx.CommonTestUtils": "1.0.0-*",
-        "Microsoft.Dnx.Runtime": "1.0.0-*",
```

Comment:
So it's technically already referenced via CommonTestUtils. I didn't mean to remove it, but I thought a hasty Ctrl-. had added it (I was a little too quick and instead of importing a namespace it seemed offer to add a reference to the package, which is odd since it should already have been referenced).

Anyway, long story short, I'll revert it.